catalog/app/containers/Bucket/PackageRevisions.js

```diff
@@ -144,141 +147,40 @@ const useRevisionStyles = M.makeStyles((t) => ({
       marginTop: t.spacing(1),
     },
   },
-  mono: {
-    fontFamily: t.typography.monospace.fontFamily,
-  },
-  time: {
-    ...t.typography.body1,
-    fontWeight: t.typography.fontWeightMedium,
-    lineHeight: t.typography.pxToRem(20),
-    whiteSpace: 'nowrap',
-  },
-  msg: {
-    WebkitBoxOrient: 'vertical',
-    WebkitLineClamp: 2,
-    display: '-webkit-box',
-    overflow: 'hidden',
-    overflowWrap: 'break-word',
-    textOverflow: 'ellipsis',
-    [t.breakpoints.up('sm')]: {
-      minHeight: 40,
-    },
-  },
-  hash: {
-    color: t.palette.text.secondary,
-    fontFamily: t.typography.monospace.fontFamily,
-    fontSize: t.typography.body2.fontSize,
-    maxWidth: 'calc(100% - 48px)',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-  },
 }))
 
-function Revision({ bucket, name, id, hash, stats, message, modified, counts }) {
-  const classes = useRevisionStyles()
-  const { urls } = NamedRoutes.use()
+function RevisionLayout({ link, msg, meta, hash, stats, counts }) {
+  const classes = useRevisionLayoutStyles()
   const t = M.useTheme()
   const xs = M.useMediaQuery(t.breakpoints.down('xs'))
   const sm = M.useMediaQuery(t.breakpoints.down('sm'))
-  // eslint-disable-next-line no-nested-ternary
```

Comment:
`no-nested-ternary` rule is still a thing. Esling shows warning on this

Reply:
yeah, i think it's ok to see those warnings and be aware of that